README.md

```diff
@@ -1 +1,123 @@
-# hotel-reservation
\ No newline at end of file
+# HOTEL-RESERVATIONS-BACK-END
+
+The backend API for the **HOTEL-RESERVATIONS-Front-End** UI, built with Ruby-on-Rails and Postgresql. The schema was setup following the ERD below.
+
+## Entity Relationship Diagram
+
+![erd](app/ERD/erd.png)
+
+## Built with
+
+- Ruby version 3.1.2
+- Rails (RoR) version 7.0.4
+- Postgresql
+- Rubocop
+
+## Front-End
+
+You can access the front-end of this application via [this link](https://github.com/Lembani/hotel-reservation-frontend.git).
+
+## SetUp
```

Comment:
- [ ] On your readme, please include the commands that are missing to run the project:`rails db:create`, `rails db:migrate` and `rails db:seed`.

That way, your users will be able to run the project 💪

Reply:
Resolved, thank you.